inc/class-manage-multisite.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+defined( 'ABSPATH' ) || die( 'File cannot be accessed directly' );
+/**
+ *
+ */
+class Manage_Multisite {
+	/**
+	 * Description
+	 *
+	 * @return type
+	 */
+	public static function init() {
+		add_action( 'admin_menu', array( __CLASS__, 'pmpro_multisite_membership_menu' ) );
+		add_action( 'admin_enqueue_scripts', array( __CLASS__, 'select_site_load_scripts' ) );
+		add_action( 'wp_ajax_select_site_get_results', array( __CLASS__, 'select_site_ajax' ) );
+		add_action( 'wp_before_admin_bar_render', array( __CLASS__, 'remove_pmpro_adminbar' ), 999 );
+	}
+
+	/**
+	 * [select_site_load_scripts description]
+	 *
+	 * @return [type] [description]
+	 */
+	public static function remove_pmpro_adminbar() {
+		global $wp_admin_bar;
+		$id = 'paid-memberships-pro';
+		$wp_admin_bar->remove_menu( $id );
+	}
+
+	/**
+	 * [select_site_load_scripts description]
+	 *
+	 * @return [type] [description]
+	 */
+	public static function select_site_load_scripts() {
+		$screen = get_current_screen();
+		if ( 'toplevel_page_pmpro-membership' === $screen->id || 'memberships_page_pmpro-multisite-membership' === $screen->id ) {
+			wp_enqueue_style( 'multisite-membership', plugin_dir_url( __FILE__ ) . 'css/multisite-membership.css' );
+		}
+		wp_enqueue_script( 'select-site', plugin_dir_url( __FILE__ ) . 'js/select-site.js', array( 'jquery' ) );
+		wp_localize_script(
+			'select-site', 'select_site_vars', array(
+				'select_site_nonce' => wp_create_nonce( 'select-site-nonce' ),
+			)
+		);
+	}
+
+	/**
+	 * [pmpro_multisite_membership_page description]
+	 *
+	 * @return [type] [description]
+	 */
+	public static function pmpro_multisite_membership_page() {
+		global $wpdb;
+		self::pmpro_membership_header();
+		?>
+		<div class="wrap">
+		<h2><?php esc_attr_e( 'PMPro Multisite Membership', 'selectsite' ); ?></h2>
```

Comment:
Should just use _e() here instead of esc_attr_e. Should also use _e()/etc around all text below.

Also the text domain should be 'pmpro-network-subsite' or whatever the slug ends up being.